components/omega/OmegaBuild.cmake

```diff
@@ -422,10 +433,10 @@ macro(init_standalone_build)
     file(APPEND ${_ProfileScript} "    -o \$OUTFILE ./src/omega.exe 1000")
 
   elseif("${OMEGA_ARCH}" STREQUAL "SYCL")
-    set(CMAKE_CXX_COMPILER ${OMEGA_SYCL_COMPILER})
+    set(CMAKE_CXX_COMPILER ${OMEGA_CXX_COMPILER})
 
-    if(OMEGA_SYCL_FLAGS)
-      set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OMEGA_SYCL_FLAGS}")
+    if(SYCL_FLAGS)
+      set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${SYCL_FLAGS}")
```

Comment:
We want to add the "OMEGA_" prefix to all Omega-related CMake variables and prefer using "OMEGA_SYCL_FLAGS" instead of "SYCL_FLAGS."